File: ProjectEuler/Problem_1/1-Multiples_of_3_and_5.cs
//Find the sum of all the multiples of 3 or 5 below 1000.

using System;

namespace Euler_1
{
    class Program
    {
        static void Main(string[] args)
        {
            int sum = 0;
            for (int i = 0; i < 1000; i++) {
                if ((i % 3 == 0) || (i % 5 == 0)) {
                sum += i;
                }
            }
            Console.WriteLine(sum);
        }
    }
}

File: ProjectEuler/Problem_1/1-Multiples_of_3_and_5.js
function Euler1(limit) {
    let total = 0;
    for (let i = 0; i < limit; i++) {
        if (i % 3 == 0 || i % 5 == 0) {
            total += i;
        }  
    }
    console.log(total); 
}

Euler1(1000);